fbw-a380x/src/wasm/systems/a380_systems/src/structural_flex.rs

```diff
@@ -0,0 +1,77 @@
+use systems::{
+    simulation::{
+        InitContext, SimulationElement, SimulationElementVisitor, SimulatorWriter, UpdateContext,
+        VariableIdentifier, Write,
+    },
+    structural_flex::elevator_flex::FlexibleElevators,
+    structural_flex::engine_wobble::EnginesFlexiblePhysics,
+    structural_flex::wing_flex::WingFlexA380,
+    structural_flex::SurfaceVibrationGenerator,
+};
+
+use uom::si::{f64::*, ratio::ratio};
+
+pub struct A380StructuralFlex {
+    ground_weight_ratio_id: VariableIdentifier,
+
+    engines_flex_physics: EnginesFlexiblePhysics<4>,
+    elevators_flex_physics: FlexibleElevators,
+    wing_flex: WingFlexA380,
+
+    surface_vibrations: SurfaceVibrationGenerator,
+}
+impl A380StructuralFlex {
+    pub fn new(context: &mut InitContext) -> Self {
+        Self {
+            ground_weight_ratio_id: context
+                .get_identifier("GROUND_WEIGHT_ON_WHEELS_RATIO".to_owned()),
+
+            engines_flex_physics: EnginesFlexiblePhysics::new(context),
+            elevators_flex_physics: FlexibleElevators::new(context),
+            wing_flex: WingFlexA380::new(context),
+
+            surface_vibrations: SurfaceVibrationGenerator::default(),
+        }
+    }
+
+    pub fn update(
+        &mut self,
+        context: &UpdateContext,
+        outter_inner_elevator_aero_torques: [(Torque, Torque); 2],
```

Comment:
```suggestion
        outer_inner_elevator_aero_torques: [(Torque, Torque); 2],
```

Typo, fix all associated usages of this param too

Reply:
God! wrong everywhere :D